### ISSUE TITLE
[WSGI] Memory leak 

### ISSUE THREAD

**Issue by remittor** [cirospaciari/socketify.py#115]
**Describe the bug**
Memory leak on testing `wrk` with many connections

**To Reproduce**
Server: Run any `"Hello World!"` test on localhost.
Client: run script:
```bash
while true
do
	wrk -t1 -c200 -d1 -H 'Connection: keep-alive' http://127.0.0.1:5000 > /dev/null
done
```

**Desktop (please complete the following information):**
 - OS: Debian 11 64-bit
 - Python: 3.9
 - Version: master branch

**Comment by cirospaciari** [comment]
Thanks for reporting I will take a look!

**Comment by remittor** [comment]
> This is WSGI, ASGI or pure socketify?

WSGI (CPython 3.9)

**Comment by cirospaciari** [comment]
It's maybe not a memory leak actually

When running in main branch on linux with the new binaries this app:

```python3
def app_hello(environ, start_response):

    start_response('200 OK', [('Content-Type', 'text/plain'), ('Content-Length', '13')])
    
    yield b'Hello, World!'
```

running this while true of work will cause memory to grow but only because python3 GC does not have a chance to run yet.

If I create an aggressive GC pass (with gc.collect()) memory will be stable.

I will investigate more, and do more tests/debugging. Thanks for reporting this memory issue.

**Comment by cirospaciari** [comment]
fixed on v0.0.19
